fedimint-ui/apps/guardian-ui/src/App.tsx

```diff
@@ -5,9 +5,11 @@ import { GuardianApi } from './GuardianApi';
 import { GuardianProvider } from './GuardianContext';
 import { Setup } from './components/Setup';
 import { formatApiErrorMessage } from './utils/api';
+import { useTranslation } from '@fedimint/translation';
 
 export const App = React.memo(function App() {
   const api = useMemo(() => new GuardianApi(), []);
+  const { t } = useTranslation('guardian');
```

Comment:
You probably want to pass in translations through the app context, `SharedChakraProvider` so you load all app translations once for the entire entire app